docs/rules/attribute-names.md

```diff
@@ -33,6 +36,55 @@ The following patterns are not warnings:
 @property({attribute: 'camel-case-name'})
 camelCaseName: string;
 
+@property({attribute: 'camel-case-other-name'})
+camelCaseName: string;
+
+@property()
+lower: string;
+```
+
+## Options
+
+You can specify `style` to one of these values `none`, `snake`, `dash` to
+enforce that attribute name is the styled version of property, or `false`.
+
+For example for a property named `camelCaseProp`, expected attribute names are:
+
+| Style | Attribute       |
+|-------|-----------------|
+| none  | camelcaseprop   |
+| dash  | camel-case-prop |
+| snake | camel_case_prop |
+
+The following patterns are considered warnings with `{"style": "dash"}`
+specified:
+
+```ts
+// Using decorators:
```

Comment:
in these examples, i'd probably just use the decorator rather than showing both each time (decorator/getter)